examples/hal/quad/main.rs

```diff
@@ -4,31 +4,33 @@
 )]
 
 extern crate env_logger;
-extern crate gfx_hal as hal;
 #[cfg(feature = "dx12")]
 extern crate gfx_backend_dx12 as back;
-#[cfg(feature = "vulkan")]
-extern crate gfx_backend_vulkan as back;
-#[cfg(feature = "metal")]
-extern crate gfx_backend_metal as back;
 #[cfg(feature = "gl")]
 extern crate gfx_backend_gl as back;
+#[cfg(feature = "metal")]
+extern crate gfx_backend_metal as back;
+#[cfg(feature = "vulkan")]
+extern crate gfx_backend_vulkan as back;
+extern crate gfx_hal as hal;
 
-extern crate winit;
+extern crate glsl_to_spirv;
 extern crate image;
+extern crate winit;
 
-use hal::{buffer, command, format as f, image as i, memory as m, pass, pso, pool};
-use hal::{Device, Instance, PhysicalDevice, Surface, Swapchain};
-use hal::{
-    DescriptorPool, FrameSync, Primitive,
-    Backbuffer, SwapchainConfig,
-};
 use hal::format::{AsFormat, ChannelType, Rgba8Srgb as ColorFormat, Swizzle};
 use hal::pass::Subpass;
 use hal::pso::{PipelineStage, ShaderStageFlags, Specialization};
 use hal::queue::Submission;
+use hal::{buffer, command, format as f, image as i, memory as m, pass, pool, pso};
+use hal::{Backbuffer, DescriptorPool, FrameSync, Primitive, SwapchainConfig};
```

Comment:
I prefer the old formatting. Any reason we need to change it here? Or is is just a rustfmt pass?
We don't want to mix formatting changes with functional one...

Reply:
Just a rustfmt pass, reverting.